packages/rum/test/userActionCollection.spec.ts

```diff
@@ -165,6 +170,32 @@ describe('newUserAction', () => {
     const userAction = spy.calls.argsFor(0)[0]
     expect(userAction.name).toBe('test-1')
   })
+
+  it('counts errors occuring during the user action', () => {
+    // tslint:disable-next-line: no-object-literal-type-assertion
+    const error = {} as ErrorMessage
```

Comment:
Would it work with
```ts
const error: ErrorMessage = {}
```

Reply:
![Screenshot 2020-04-14 at 10 31 18](https://user-images.githubusercontent.com/61787/79203435-5441f380-7e3b-11ea-98cd-1882e80f29de.png)
`const errorMessage = {} as unknown as ErrorMessage;` works, but IMO that's not better than disabling the tslint rule.